doc/reference/task_wdt/index.rst

```diff
@@ -0,0 +1,53 @@
+.. _task_wdt:
+
+Task Watchdog
+#############
+
+Overview
+********
+
+Many microcontrollers feature a hardware watchdog timer peripheral. Its purpose
+is to trigger an action (usually a system reset) in case of severe software
+malfunctions. Once initialized, the watchdog timer has to be restarted ("fed")
```

Comment:
This "fed" means "feed"?

Reply:
"fed" is meant an alternative term for "restarted", so I used the past participle form as well. I think it is correct.

Reply:
Ah, I see. Thanks.